targets/xorg

```diff
@@ -181,7 +181,7 @@ if [ -n "$backport" ]; then
         "xserver-xorg-input-synaptics$backport" $installvideodrivers
 fi
 
-if release -lt stretch -lt kali-rolling -lt yakkety; then
+if release -lt stretch -lt kali-rolling -lt yakkety -lt tumbleweed; then
```

Comment:
note that if you don't want this for any opensuse release, you can just leave the line unmodified (without `-lt tumbleweed`) and it won't match opensuse.